zndraw/utils.py

```diff
@@ -23,6 +24,12 @@ def split_list_into_chunks(lst, n):
         yield lst[i : i + n]
 
 
+def estimate_max_batch_size_for_socket(frames: list[znframe.Frame], max_size=1e6):
+    sizes = [sys.getsizeof(frame) for frame in frames]
+    largest_frame = max(sizes)
+    return int(max_size / largest_frame * 0.9)
```

Comment:
we should get the `max_size` from the config e.g. if we want to adapt the socketio settings

Reply:
Sure, I assume we also need to configure the socket somewhere in the `app.py` for this as well